Source/NaturalLanguageClassifierV1/NaturalLanguageClassifier.swift

```diff
@@ -0,0 +1,239 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Alamofire
+import Freddy
+
+public class NaturalLanguageClassifierV1 {
+    
+    private let username: String
+    private let password: String
+    
+    private let domain = "com.ibm.watson.developer-cloud.WatsonDeveloperCloud"
+    private let serviceURL = "https://gateway.watsonplatform.net/natural-language-classifier/api"
+    
+    public init(username: String, password: String) {
+        self.username = username
+        self.password = password
+    }
+    
+    private func dataToError(data: NSData) -> NSError? {
+        do {
+            let json = try JSON(data: data)
+            let error = try json.string("error")
+            let code = try json.int("code")
+            let description = try json.string("description")
+            let userInfo = [
+                NSLocalizedFailureReasonErrorKey: error,
+                NSLocalizedRecoverySuggestionErrorKey: description
+            ]
+            return NSError(domain: domain, code: code, userInfo: userInfo)
+        } catch {
+            return nil
+        }
+    }
+    
+    /**
+     Retrieves the list of classifiers for the service instance. Returns an empty array
+     if no classifiers are available.
+     
+     - parameter failure: A function executed if an error occurs.
+     - parameter success: A function executed with the list of available standard and custom models.
+     */
+    public func getClassifiers(
+        failure: (NSError -> Void)? = nil,
+        success: [ClassifierModel] -> Void) {
+        
+        // construct REST request
+        let request = RestRequest(
+            method: .GET,
+            url: serviceURL + "/v1/classifiers",
+            acceptType: "application/json"
+        )
+        
+        // execute REST request
+        Alamofire.request(request)
+            .authenticate(user: username, password: password)
+            .responseArray(dataToError: dataToError, path: ["classifiers"]) {
+                (response: Response<[ClassifierModel], NSError>) in
+                switch response.result {
+                case .Success(let classifiers): success(classifiers)
+                case .Failure(let error): failure?(error)
+                }
+        }
+    }
+    
+    /**
+     Sends data to create and train a classifier and returns information about the new
+     classifier. When the operation is successful, the status of the classifier is set
+     to "Training". The status must be "Available" before you can use the classifier.
+     
+     - parameter trainingMetadata:
+     - parameter trainingData:      The set of questions and their "keys" used to adapt a system to a domain (the ground truth).
+     - parameter failure: A function executed if an error occurs.
+     - parameter success: A function executed with the list of available standard and custom models.
```

Comment:
This comment should probably be updated. We should add a description for `trainingMetadata`, remove the whitespace from `trainingData`, and I think the `success` parameter needs to changed as well.

I think this is actually the hardest part of the SDK--writing good, descriptive, and up-to-date documentation. (The Swagger documentation doesn't always help, either...)